Lib/test/test_capi/test_hash.py

```diff
@@ -77,3 +77,13 @@ def python_hash_pointer(x):
         # Py_HashPointer((void*)(uintptr_t)-1) doesn't return -1 but -2
         VOID_P_MAX = -1 & (2 ** (8 * SIZEOF_VOID_P) - 1)
         self.assertEqual(hash_pointer(VOID_P_MAX), -2)
+
+    def test_hash_bytes(self):
+        hash_bytes = _testcapi.hash_bytes
+
+        def check(data):
```

Comment:
Should we add a test where the input is of incorrect type?

Reply:
`Py_HashBytes(const void *ptr, Py_ssize_t len)` is a C API which accepts any memory, it has no Python type.

Reply:
Oh right. Sorry, I think I got confused by

```rst
The hash is guaranteed to match that of
:class:`bytes`, :class:`memoryview`, and other built-in objects
that implement the :ref:`buffer protocol <bufferobjects>`.
```

(The fault lies with me, the docs have no issues).